src/components/Routes/Home.tsx

```diff
@@ -20,10 +20,9 @@ const Home = () => {
     return null
   }
 
-  // Get the name, or if it does not exist in the traits, use the
-  // identity's ID
-  const { name: { first = String(session.identity.id) } = {} } = session
-    .identity.traits as any
+  const traits = session.identity.traits
```

Comment:
isn't identity optional now? 
```suggestion
  const traits = session.identity?.traits
```